src/model_theory/basic.lean

```diff
@@ -105,6 +106,10 @@ instance is_relational_sum [L.is_relational] [L'.is_relational] : is_relational
 instance is_algebraic_sum [L.is_algebraic] [L'.is_algebraic] : is_algebraic (L.sum L') :=
 ⟨λ n, sum.is_empty⟩
 
+instance is_relational.countable_functions [L.is_relational] :
+  nonempty (encodable (Σl, L.functions l)) :=
```

Comment:
I don't think we should make global instances for the existence of some structure on some type. That would explode quickly. If you need this occasionally, you can use `haveI` every time. If you need this a lot, it can be a `localized` instance. This can become an instance if we define a separate typeclass `countable`, but now I don't think it's good.

Reply:
I've added some classes specifically for countability of languages to replace this.